cli.ts

```diff
@@ -1,18 +1,18 @@
-import { resolve } from 'https://deno.land/std@0.96.0/path/mod.ts'
-import { parse } from 'https://deno.land/std@0.96.0/flags/mod.ts'
-import { existsDir } from './shared/fs.ts'
-import log, { LevelNames } from './shared/log.ts'
-import util from './shared/util.ts'
-import { VERSION } from './version.ts'
+import { parse } from "https://deno.land/std@0.96.0/flags/mod.ts";
```

Comment:
pls fix the format we are following:
- single quote
- no semicolon end of the line

Reply:
yes , sorry i run `deno fmt`

Reply:
cool

Reply:
@ije done. can you pls review changes.

Reply:
sorry i can not see the changes, still double quotes

Reply:
@ije fixed